cmd/operator-sdk/test/local.go

```diff
@@ -68,7 +69,8 @@ func newTestLocalCmd() *cobra.Command {
 	testCmd.Flags().StringVar(&tlConfig.namespacedManPath, "namespaced-manifest", "", "Path to manifest for per-test, namespaced resources (e.g. RBAC and Operator manifest)")
 	testCmd.Flags().StringVar(&tlConfig.goTestFlags, "go-test-flags", "", "Additional flags to pass to go test")
 	testCmd.Flags().StringVar(&tlConfig.moleculeTestFlags, "molecule-test-flags", "", "Additional flags to pass to molecule test")
-	testCmd.Flags().StringVar(&tlConfig.namespace, "namespace", "", "If non-empty, single namespace to run tests in")
+	testCmd.Flags().StringVar(&tlConfig.deployNamespace, "deploy-namespace", "", "If non-empty, single namespace to run tests in (deploys namespaced resources here); default: \"\" which resolves to namespace from kubeconfig")
+	testCmd.Flags().StringVar(&tlConfig.watchNamespace, "watch-namespace", "", "The namespace where the operator watches for changes (only valid with --up-local). Explicitly set to empty string to watch all namespaces (defaults to the deploy namespace).")
```

Comment:
```suggestion
	testCmd.Flags().StringVar(&tlConfig.watchNamespace, "watch-namespace", "", "(only valid with --up-local) Define the operator scope. To watch all namespaces use an empty string as \"\"(By default it will be the operator namespace).")
```

Reply:
I like the idea of moving `(only valid with --up-local)` to the beginning to the help text. Otherwise, I think the original text is more clear.